index.html

```diff
@@ -71,5 +71,87 @@ <h2>Software Developer</h2>
 
        </div>
    </section>
+
```

Comment:
Good work, following HTML best practices using HTML semantics 😎 
which will defiantly enhance your SEO ranking. 💹 

_You could add also `<aside>` , `<article>` and `<summery>` to your collection ➕_